src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -927,6 +927,20 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
 
 "ExposureSubmission_OverwriteNotice_Antigen_Text" = "Sie haben bereits einen Schnelltest registriert. Die App kann maximal einen Schnelltest und einen PCR-Test gleichzeitig verwalten. Wenn Sie einen weiteren Schnelltest registrieren, wird der erste Schnelltest aus der App gelöscht.";
 
+/* ExposureSubmission AntigenTest Profile Information */
+
+"ExposureSubmission_Antigen_Profile_Information_Title" = "Schnelltest-Profil";
+
+"ExposureSubmission_Antigen_Profile_Information_Image_Description" = "Eine QR Code der eine Person auf dem Smartphone identifiziert.";
```

Comment:
German should be "QR-Code" with a hyphen. 
"Eine QR-Code" sounds strange, I would say "Ein QR-Code".

```suggestion
"ExposureSubmission_Antigen_Profile_Information_Image_Description" = "Ein QR-Code der eine Person auf dem Smartphone identifiziert.";
```

@janetback can you confirm?

Reply:
Disregard this, @janetback made another proposal [here](https://github.com/corona-warn-app/cwa-app-ios/pull/2577#pullrequestreview-647938787)